reference/6/Microsoft.PowerShell.Core/About/about_PowerShell_Config.md

```diff
@@ -0,0 +1,350 @@
+```Schema
+"LogKeywords": "<log-keyword>,..."
+```
+
+Where:
+
+- `<log-keyword>` is one of:
+  - Runspace - runspace management
+  - Pipeline - pipeline operations
+  - Protocol - communication protocol handling, such as PSRP
+  - Transport - transport layer support, such as SSH
+  - Host - PowerShell host functionality, for example console interaction
+  - Cmdlets -PowerShell builtin cmdlets
+  - Serializer - serialization logic
+  - Session - PowerShell session state
+  - ManagedPlugin - WSMan plugin
+
+> [!NOTE]
+> It is generally advised to leave this value unset
+> unless you are trying to diagnose a specific behavior
+> in a known part of PowerShell.
+> Changing this value only decreases the amount of information logged.
+
+This example limits the logging to runspace operations, pipeline logic, and
+cmdlet use. All other logging will be omitted.
+
+```json
+"LogKeywords": "Runspace,Pipeline,Cmdlets"
+```
+
+<!--
+## Group policy configuration
+
+### ScriptExecution
+
+### ScriptBlockLogging
+
+### ProtectedEventLogging
+
+### Transcription
+
+### UpdateableHelp
+
+### ConsoleSessionConfiguration
+-->
+
+## More example configurations
+
+### Example Windows configuration
+
+This configuration has more settings explicitly set:
+
+- Execution policy for this PowerShell installation is `AllSigned`
+- The module path is set to include three module directories
+- The `PSImplicitRemotingBatching` experimental feature is enabled
+
+> [!NOTE]
+> The `ExecutionPolicy` and `PSModulePath` settings here would only work on a Windows platform,
+> since the module path uses `\` and `;` separator characters
+> and the execution policy is not `Unrestricted` (the only policy allowed on UNIX-like platforms).
+
+```json
+{
+  "Microsoft.PowerShell.ExecutionPolicy": "AllSigned",
+  "PSModulePath": "C:\\Users\\Marisol Briggs\\Documents\\PowerShell\\Modules;C:\\Shared\\Modules;C:\\Program Files\\PowerShell\\6\\Modules",
+  "ExperimentalFeatures": ["PSImplicitRemotingBatching"],
+}
+```
+
+### Example non-Windows configuration
+
+This configuration sets a number of options that only work in macOS or Linux:
+
+- The module path contains three module directories
+- The **PSImplicitRemotingBatching** experimental feature is enabled
+- The PowerShell logging level is set to **Verbose**, for more logging
+- This PowerShell installation writes to the logs using the **home-powershell**
+  identity.
+
+
+```json
+{
+  "PSModulePath": "/home/bmcnamara/.config/powershell/Modules:/mnt/sparedrive/PowerShellModules:/opt/microsoft/powershell/Modules",
+  "ExperimentalFeatures": ["PSImplicitRemotingBatching"],
+  "LogLevel": "Verbose",
+  "LogIdentity": "home-powershell"
+}
+```
+
+## See also
+
+[About Execution Policies](./about_Execution_Policies.md)
+
+[About Automatic Variables](./about_Automatic_Variables.md)
+
+[RFC0029]: https://github.com/PowerShell/PowerShell-RFC/blob/master/5-Final/RFC0029-Support-Experimental-Features.md
```

Comment:
```suggestion
[Experimental Features - RFC0029]: https://github.com/PowerShell/PowerShell-RFC/blob/master/5-Final/RFC0029-Support-Experimental-Features.md
```

Reply:
This is a reference link. Your change breaks the reference.

Reply:
I can change the reference everywhere